README.md

```diff
@@ -220,12 +220,12 @@ Users can also change the configuration parameters of NanoMQ while booting. Howe
 
 In the `etc` directory of NanoMQ (`./nanomq/etc/nanomq.conf`), there is a 'nanomq.conf' configuration file. This file is different from 'config.cmake.in'. This 'nanomq.conf' allows you to configure broker when booting. Please be noted that if you start NanoMQ in the project's root directory, this file will be read automatically.
```

Comment:
now we have moved the default configuration file path to /etc/nanomq*.conf

Reply:
NanoMQ will look up to its configuration file in "/etc/" by default. Please remember to copy conf file to /etc/ in your system if you wanna start NanoMQ without setting conf path manually.
replace 
In the `etc` directory of NanoMQ (`./nanomq/etc/nanomq.conf`), there is a 'nanomq.conf' configuration file.